README.md

```diff
@@ -1,24 +1,144 @@
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/HSMathebula/movie-booking-app-back-end.git
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+### Run locally
+
+Make your way to the correct directory by running this command:
+
+```
+$ cd movie-booking-app
+```
+
+Install the required dependencies to run the project with this command:
+
+```
+$ bundle install
+```
+
+Setup the database, run:
+
+```
+$ rails db:create
+$ rails db:migrate
+$ rails db:seed
+```
+
+Then run it in your browser with this command:
+
+```
+$ rails server
+```
+
+### Run tests
+
+To run tests, please run this command:
+
+```
+$ rspec
+```
+
+## Authors
+
+👤 Ilham Bouaik <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [GitHub](https://github.com/BouaikIlham)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [Twitter](https://twitter.com/IlhamBouaik)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [LinkedIn](https://www.linkedin.com/in/bouaik-ilham-478478230/)
+
+👤 Haji Bamsi <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [Github](https://github.com/bamsi)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [Twitter](https://twitter.com/haji-bamsi-17327728/)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [LinkedIn](https://linkedin.com/in/bamsi)
+
+👤 Happen Mathebula <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [Github](https://github.com/HSMathebula)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [Twitter](https://twitter.com/HappenSipho)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [LinkedIn](www.linkedin.com/in/happen-sipho-mathebula)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://www.microverse.org/)
+
+## 📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] Please, go [here](https://github.com/microverseinc/readme-template) and grab the `MIT.md` file and add it to the root of your project, so it has a license.  Then, kindly link to it in the "License" section of your readme.

That way, your project will have a license, and a link to it 💪